sentinel/tests/unit/schedule/reminders.js

```diff
@@ -113,49 +112,6 @@ describe('reminders', () => {
     });
   });
 
-  describe('getSchedule', () => {
-    let getSchedule;
-    beforeEach(() => {
-      sinon.stub(later, 'schedule');
-      sinon.stub(later.parse, 'text');
-      sinon.stub(later.parse, 'cron');
-      getSchedule = reminders.__get__('getSchedule');
-    });
-
-    it('should return schedule for cron', () => {
-      later.parse.cron.returns('parsed cron');
-      later.schedule.returns('schedule');
-      assert.equal(getSchedule({ cron: 'something' }), 'schedule');
-      assert.equal(later.parse.cron.callCount, 1);
-      assert.deepEqual(later.parse.cron.args[0], ['something']);
-      assert.equal(later.parse.text.callCount, 0);
-      assert.equal(later.schedule.callCount, 1);
-      assert.deepEqual(later.schedule.args[0], ['parsed cron']);
-    });
-
-    it('should return schedule for text_expression', () => {
-      later.parse.text.returns('parsed expression');
-      later.schedule.returns('schedule');
-      assert.equal(getSchedule({ text_expression: 'other' }), 'schedule');
-      assert.equal(later.parse.text.callCount, 1);
-      assert.deepEqual(later.parse.text.args[0], ['other']);
-      assert.equal(later.parse.cron.callCount, 0);
-      assert.equal(later.schedule.callCount, 1);
-      assert.deepEqual(later.schedule.args[0], ['parsed expression']);
-    });
-
-    it('should prioritize text_expression', () => {
```

Comment:
I don't follow what you want to change, this test and the other tests within the same `describe` block were moved from [here](https://github.com/medic/cht-core/blob/8f328d40b21dd95a6f24606e19c607c75e8dc41d/sentinel/tests/unit/schedule/reminders.js#L147), because I refactored the `getSchedule` moving it to a common module to be reused in another module when the same code was repeated, but with a different return so I needed to change the tests a bit, but the logic tested is the same.

Reply:
You removed the tests around getting the schedule, and in the actual code you added this to replace it: 
```
return later.schedule(scheduling.getSchedule(config));
```

I'd like a test that checks how this function is called.